geograypher/constants.py

```diff
@@ -138,4 +138,4 @@ def hex_to_rgb(value):
     "TSME",
 )
 
-DEFAULT_FRUSTUM_SCALE = 1
+DEFAULT_FRUSTUM_SCALE = 1
```

Comment:
I thought black ensured a trailing newline but it appears to not. Since there's nothing saying this is wrong, I'll merge this now. However, in the future, I think we should pick a convention, update everything to match, and then stick to it.